src/HexagonOptimize.cpp

```diff
@@ -679,6 +681,27 @@ class OptimizePatterns : public IRMutator {
                 expr = mutate(op->a + neg_b);
                 return;
             } else {
+                // No widening subtracts for the expressions of the form:
```

Comment:
(the relevance of the BTW being that find_mpy_ops might go away or change form significantly.)